java/src/Ice/src/main/java/com/zeroc/Ice/ReplyStatus.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) ZeroC, Inc.
+
+package com.zeroc.Ice;
+
+public enum ReplyStatus {
+  Ok(0),
+  UserException(1),
+  ObjectNotExist(2),
+  FacetNotExist(3),
+  OperationNotExist(4),
+  UnknownLocalException(5),
+  UnknownUserException(6),
+  UnknownException(7);
+
+  private final byte value;
+
+  private ReplyStatus(int value) {
+    this.value = (byte) value;
+  }
+
+  public byte getValue() {
+    return value;
+  }
```

Comment:
This is almost certainly a matter of opinion (and a divided one in Java),
but I feel like it's completely fine to use a `public final` field here, instead of implementing a getter.

Getters make some sense for encapsulation of classes, but for such a simple enum, I don't see the point.
It's not like these values will ever change, or we'll need to do any additional processing of them.

These random people on the internet agree with me: https://softwareengineering.stackexchange.com/questions/275691/is-public-final-completely-not-ok-in-constant-enum-fields

Reply:
We should be consistent with the Slice-to-Java mapping, even though it's not a Slice-define enum.

With the Slice-to-Java mapping, we generate a `value()` method:
https://doc.zeroc.com/ice/3.7/language-mappings/java-mapping/client-side-slice-to-java-mapping/java-mapping-for-enumerations

Will fix.

Reply:
Fixed.